justfile

```diff
@@ -11,6 +11,65 @@ check-format:
     just --unstable --fmt --check -f justfile
     yarn prettier --no-error-on-unmatched-pattern --check **/*.yml **/*.json **/*.md
 
+# Generates the daemonset manifest for kube-vip.
+[group('codegen')]
+codegen-kube-vip-daemonset:
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # renovate: datasource=docker depName=ghcr.io/kube-vip/kube-vip
+    KUBE_VIP_VERSION=v0.8.10
+    DEST=kustomization/components/kube-vip/daemonset.yml
+    SCRATCH=/tmp/daemonset.yml
```

Comment:
nit: should probably use `mktemp` to create this path

Reply:
Probably.  Can do that in the future if we mess around with this more (which I will for the upgrade and splitting, alas)